optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -43,12 +46,15 @@
         "incrementalScoreCalculatorCustomProperties",
         "scoreDrlList",
         "scoreDrlFileList",
+        "compileDroolsAlphaNetwork",
         "kieBaseConfigurationProperties",
         "initializingScoreTrend",
         "assertionScoreDirectorFactory"
 })
 public class ScoreDirectorFactoryConfig extends AbstractConfig<ScoreDirectorFactoryConfig> {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScoreDirectorFactoryConfig.class);
```

Comment:
(separate issue)
in SolverManagerConfig we use the variable name `logger`, which violates name conventions, lets' fix this accross the code in a different PR?